build.gradle

```diff
@@ -367,4 +367,6 @@ task verifyLicense << {
   }
 }
 test.dependsOn verifyLicense
-test.dependsOn verifyGoogleJavaFormat
+
+// TODO(pongad): Unflake the build
```

Comment:
add a little description that this races with AutoValue generation (which does not comply with the formatting standards enforced here)